ExamplePlugins/ExampleCommandLineInterfaceTool/README.md

```diff
@@ -0,0 +1,34 @@
+﻿# Extending the LabVIEW NXG CLI with custom tools
+
+The LabVIEW NXG CLI (`labviewnxgcli.exe`) supports custom tools that are exported via MEF. 
+
```

Comment:
Can you add a Requirements section or specify that the CLI exists in LabVIEW NXG 3.1 and later?

Reply:
Updated this to "**Introduced in LabVIEW NXG 3.1**, the LabVIEW NXG CLI (`labviewnxgcli.exe`) supports custom tools that are exported via MEF. "